fiks-meldingsformidler/src/main/java/no/difi/meldingsutveksling/ks/mapping/FiksStatusMapper.java

```diff
@@ -0,0 +1,52 @@
+package no.difi.meldingsutveksling.ks.mapping;
+
+import lombok.RequiredArgsConstructor;
+import no.difi.meldingsutveksling.ks.svarut.ForsendelseStatus;
+import no.difi.meldingsutveksling.receipt.MessageStatus;
+import no.difi.meldingsutveksling.receipt.MessageStatusFactory;
+import no.difi.meldingsutveksling.receipt.ReceiptStatus;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class FiksStatusMapper {
+
+    private final MessageStatusFactory messageStatusFactory;
+
+    public MessageStatus mapFrom(ForsendelseStatus forsendelseStatus) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The Cyclomatic Complexity of this method "mapFrom" is 15 which is greater than 10 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)